privacy-policy/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: Privacy Policy
+---
+
+# Vox Pupuli Privacy Policy
+
+This document details the data the Vox Pupuli collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Vox Pupuli project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * E-Mail address
+* Data that identifies you
+  * IP Address
+  * Browser UserAgent Strings
+
+# How and why we use your data
+
+## The website
+
+The Vox Pupuli website is a static site that does not track, collect, or store 
+infromation or set cookies. There is no account needed or required to access 
+the website itself. The website itself is hosted with [Github Pages](https://pages.github.com/) 
+and any data gathered by Github will adhere to the [Github Privacy Policy](https://help.github.com/articles/github-privacy-statement/).
+
```

Comment:
In addition we should add a note about cloudflare. They provide https support and store (2?) cookies on each device.

Reply:
Since github now provides custom domains with HTTPS (https://blog.github.com/2018-05-01-github-pages-custom-domains-https/) I wonder if we should move away from cloudflare to use as little services as possible.

Reply:
CloudFlare is now mentioned and added to the Privacy Policy links at the bottom of the document.